database/pre_cmd.sql

```diff
@@ -0,0 +1 @@
+PRAGMA foreign_keys = 'ON';
```

Comment:
This option ensures that the foreign key constraints we have set are correctly enforced. This prevents the addition of data that does not conform to our database structure and also prevents data corruption.
More info: https://www.sqlite.org/foreignkeys.html

Reply:
I decided to keep this as a separate file because it could be the case that we implement features that depend on more of these types of commands in the future. I'm just trying to future proof this.